sopel/loader.py

```diff
@@ -116,6 +116,9 @@ def compile_rule(nick, pattern):
         return pattern
 
     nick = re.escape(nick)
+    botflt = re.match('(?i)(.*)([-_.]?bot$)', nick)
```

Comment:
Why not name it `bot_filter` then? It's much easier to read, and there's no penalty for long variable names in Python.